parallel/20200722/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
```

Comment:
`LABEL version="1"` --> `LABEL dockerfile.version="1"`

Reply:
I'll get that right eventually